### PR TITLE
[master] Bug 536853: Issue with processing MapsId with complex composite and d…

### PR DESCRIPTION
**Pull request by jgrassel**
Pull request for master associated with https://bugs.eclipse.org/bugs/show_bug.cgi?id=536853

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=536853

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=536853

File: jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ * Contributors:
+ *     07/09/2018-2.6 Jody Grassel
+ *       - 536853: MapsID processing sets up to fail validation
+ ******************************************************************************/
```
Comment (lukasj) [inline jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java]: glassfish-copyright-plugin will fail to verify this. To void another "batch change" once we add it to the build, I'd suggest to follow the pattern used in other files (or shown in https://github.com/eclipse-ee4j/eclipselink/blob/master/project-admin/Sample.java)

File: jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
+/********************************************************************************
+ * Copyright (c) 1998, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1998, 2018 IBM Corporation. All rights reserved.
```
Comment (lukasj) [inline jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java]: In new files copyright year should be 2018 only and copyright holder should the one submitting the PR - so in this case you. In the other words - first line should be removed and second line should not contain 1998.

File: jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java (hunk 1 above)
Comment (jgrassel) [reply jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java]: @lukasj Fixed

File: jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java (hunk 2 above)
Comment (jgrassel) [reply jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java]: @lukasj

File: jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java (hunk 1 above)
Comment (jgrassel) [reply jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java]: Fixed in 2.7 pull request, updating the one for master now.

**Pull request merged**